MovieChecker/MovieChecker/BoxOfficeCell.m

```diff
@@ -0,0 +1,25 @@
+//
+//  BoxOfficeCell.m
+//  MovieChecker
+//
+//  Created by Joseph Goldberg on 1/22/15.
+//  Copyright (c) 2015 Vokal. All rights reserved.
+//
+
+#import "BoxOfficeCell.h"
+
+@implementation BoxOfficeCell
+
+- (void)awakeFromNib
+{
+    // Initialization code
+}
```

Comment:
1. On iOS, you should always call `[super awakeFromNib]` in any implementation of `awakeFromNib`.
2. Unless there's a reason to keep it, delete empty methods; if there's a reason to keep it, document it with a comment.